### ISSUE TITLE
Enhancement: Hyper-yes style workaround for `Terminate Batch Job?` prompt

### ISSUE THREAD

**Issue by mikemaccana** [felixse/FluentTerminal#313]
Ideally, people shouldn't be making or using `cmd` scripts in 2019. But just in case they are, Hyper has a useful plugin called [hyper-yes](https://github.com/albertjan/hyper-yes) that automatically responds:

![hyper-yes](https://user-images.githubusercontent.com/172594/55826827-bec1a480-5b00-11e9-8873-8299c34577ff.gif)

Maybe this could be useful for Fluent? Maybe it could be a plugin (if those exist) instead. Just a thought.

**Comment by skipmeister123** [comment]
We haven't talked about plugins, but it's something I've personally been thinking about lately.  Specifically, I've been eyeing Terminus (https://eugeny.github.io/terminus/).  They've implemented several features which we sorely lack, including plugins.

**Comment by felixse** [comment]
Yeah, plugins are a bit easier to support when you're based on JavaScript 😄 

The feature looks quite useful, I have no problems with adding something like this directly (optional of course) to the application

**Comment by AndrewKralovec** [comment]
Agreed, plugins are easier in JavaScript because of the prototype inheritance. 

I suppose we can add a plugins panel in the settings. Let you specify the path of the dll and work with a interface that we can inject. However, were are we going to be calling the plugins ? Seems kinda messy.
I have see the use of [MEF](https://archive.codeplex.com/?p=mef), in other projects, but the documentation is terrible.
